src/mono/mono.proj

```diff
@@ -933,16 +938,52 @@
     <Error Condition="!Exists('$(LocalDotnet)')" Text="'$(LocalDotnet)' doesn't exist." />
   </Target>
 
+  <Target Name="BuildMonoRuntimeCMake" Condition="'$(MonoEnableCMake)' == 'true'">
```

Comment:
Looks specific to a couple of platform builds at the moment (osx, browser), should we have an error if you try to enable cmake build for a platform that is currently not supported?

Reply:
It's not supported on any platforms right now.